telematic_system/scripts/log_analysis/get_message_drop.py

```diff
@@ -38,12 +38,12 @@ def combineFiles(log_dir):
     bridges_csv = []
 
     messaging_server_csv_exist = False
-    messaging_server_csv = ""
+    messaging_server_csv = []
 
     for filename in filenames:        
-        if "Messaging" in filename:
+        if "messaging".lower() in filename:
```

Comment:
Can we change this to `if "messaging" in filename.lower()` so that we're not expected "Messaging"